scene/gui/reference_rect.cpp

```diff
@@ -38,26 +38,41 @@ void ReferenceRect::_notification(int p_what) {
 
 		if (!is_inside_tree())
 			return;
-		if (Engine::get_singleton()->is_editor_hint())
+		if (Engine::get_singleton()->is_editor_hint() || !editor_only)
 			draw_rect(Rect2(Point2(), get_size()), border_color, false);
 	}
 }
 
 void ReferenceRect::set_border_color(const Color &color) {
 	border_color = color;
+	update();
 }
 
 Color ReferenceRect::get_border_color() const {
 	return border_color;
 }
 
+void ReferenceRect::set_editor_only(const bool &enabled) {
```

Comment:
Ah wait, this should be p_enabled (same in the `.h` file), to match most functions in the source code. While at it, you might change `set_border_color`'s parameter's name as well.

Reply:
`ClassDB::bind_method(D_METHOD("set_editor_only", "enabled"), &ReferenceRect::set_editor_only);`
Sure! I have to change the `enabled` parameter in the bind_method or is this just a reference name for the real parameter?

Reply:
Well, in the C++ code, we use `p_xxx` for parameter names, and `r_xxx` for passed-by-reference parameters which are modified within the function. So, only in the code, not in the bind.